### PR TITLE
tests: add missing arrays to InspectorIssues sample artifact

### PR DESCRIPTION
**Pull request by connorjclark**
This is the error w/o this change:

> inspector-issues:warn Caught exception: Cannot read property 'length' of undefined

### CODE REVIEW

File: lighthouse-core/test/results/artifacts/defaultPass.trace.json [hunk 1]
```diff
@@ -14505,7 +14505,7 @@
   {"pid":75994,"tid":17667,"ts":185608246435,"ph":"X","cat":"toplevel","name":"MessageLoop::RunTask","args":{"src_file":"../../ipc/ipc_mojo_bootstrap.cc","src_func":"SendMessage"},"dur":31,"tdur":30,"tts":78713},
   {"pid":75994,"tid":17667,"ts":185608246474,"ph":"X","cat":"toplevel","name":"MessageLoop::RunTask","args":{"src_file":"../../ipc/ipc_mojo_bootstrap.cc","src_func":"SendMessage"},"dur":16,"tdur":16,"tts":78752},
   {"pid":75994,"tid":17667,"ts":185608247189,"ph":"X","cat":"toplevel","name":"MessageLoop::RunTask","args":{"src_file":"../../ipc/ipc_mojo_bootstrap.cc","src_func":"SendMessage"},"dur":30,"tdur":29,"tts":78797},
-  {"_comment":"Manually added event to make sample lhr not error","name":"largestContentfulPaint::Candidate","pid":75994,"tid":17667,"ts":185608247190,"ph":"R","cat":"loading,rail,devtools.timeline","args":{"frame":"0x44d2861df8", "data": {"size": 50}}},
+  {"_comment":"Manually added event to make sample lhr not error","name":"largestContentfulPaint::Candidate","pid":75994,"tid":17667,"ts":185608247190,"ph":"R","cat":"loading,rail,devtools.timeline","args":{"frame":"0x44d2861df8","data":{"size":50}}},
```
Comment (connorjclark) [inline lighthouse-core/test/results/artifacts/defaultPass.trace.json:14508]: this is gonna keep happening until we commit this (whitespace) change.

File: lighthouse-core/test/results/sample_v2.json [hunk 2]
```diff
@@ -5532,7 +5536,7 @@
         }
       ],
       "id": "best-practices",
-      "score": null
+      "score": 0.13
```
Comment (connorjclark) [inline lighthouse-core/test/results/sample_v2.json:5539]: 👀

**Comment by brendankenny** [comment]
wow we really should have noticed an errored Best Practices in sample_v2 in the last month of PRs...

**Comment by connorjclark** [comment]
Agreed, let's add a test.

**Comment by connorjclark** [comment]
hmm, we don't have any jest tests that do assertions on the sample json, so not sure where to put it. wdyt? `runner-test`? or a new file `sample-json-test`?

**Comment by paulirish** [comment]
>  new file `sample-json-test`?

👍

**Review by Beytoven — APPROVED** [review]
Yikes, can't believe I missed this. Great fix! 👍

**Pull request merged**